tests/integration/Utils.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenStack\Integration;
+
+use GuzzleHttp\Client;
+use OpenStack\Identity\v2\Api;
+use OpenStack\Identity\v2\Service;
+use OpenStack\Common\Transport\HandlerStack;
+
+class Utils
+{
+    public static function getAuthOptsV3()
+    {
+        return [
+            'authUrl' => getenv('OS_AUTH_URL'),
+            'region'  => getenv('OS_REGION_NAME'),
+            'user'    => [
+                'id'       => getenv('OS_USER_ID'),
+                'password' => getenv('OS_PASSWORD'),
+            ],
+            'scope'   => [
+                'project' => [
+                    'id' => getenv('OS_PROJECT_ID'),
+                ]
+            ]
+        ];
+    }
+
+    public static function getAuthOptsV2()
+    {
+        $authUrl = \OpenStack\Common\Transport\Utils::normalizeUrl(getenv('OS_AUTH_URL'));
```

Comment:
I guess it is a different Utils.php not the one under Transports. Would you like to move the auth under Transport Utils or clone the normalizeUrl into this class?